src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -100,6 +100,21 @@
               </v-btn>
             </div>
           </v-col>
+          <v-col cols="1" v-else-if="isSpecialResolutionFiling">
+            <div class="d-flex justify-end align-end align-sm-start tooltip-info">
```

Comment:
How about moving this class to the tooltip itself as you did in ChangeBusinessType.vue?

Reply:
class for the tooltip is not aligning properly, (seems like the tooltip class has flex-align-center)
![Screen Shot 2022-07-28 at 11 36 14 AM](https://user-images.githubusercontent.com/55506249/181613355-0def9652-5c04-4fb1-8296-eacc9bf8d333.png)

when we give class for div its aligning properly
![Screen Shot 2022-07-28 at 11 39 44 AM](https://user-images.githubusercontent.com/55506249/181613421-4dcd4d1f-a479-46af-9b5c-ccfdfd146fca.png)

Reply:
Why does the one in ChangeBusinessType.vue work? I'm hoping for consistency.

Reply:
oh, There we don't need to align to the end, it is next to the text, so we don't need additional class there. Here we need to show at the end of the column
we can use like in the change button (actions class). but it has a minus margin to fix button, so we cant use it
![Screen Shot 2022-07-28 at 11 50 22 AM](https://user-images.githubusercontent.com/55506249/181615853-67e2d607-b91d-4bdd-a3bb-731abf585c0f.png)

Reply:
What bugs me here is that we have the same classname used for 2 different types of elements.

How about renaming this one "tooltip-wrapper" or "tooltip-container"?

Alternately, you could delete the classname from both (since you don't use the classname to set CSS properties) and then in your unit tests you can select the elements using other ways (for example, the Vuetify tootip component has its own class names already).

Up to you, but this bugs me a little :)

Reply:
Agreed, removed extra class and use vutify class for testing :)

Reply:
updated , shall i merge it ?

Reply:
Looks great! Yes, please merge this.